Feat never merge

bemble opened the pull request:
As asked in #37, there's a logic around spool merging, depending on spools used filament etc, even if a tag is present.

So, that kind of situation happens:
<img width="1458" alt="Capture d’écran 2025-03-17 à 00 37 00" src="https://github.com/user-attachments/assets/db4102a7-5dea-4a9e-a0e6-fd519cac1fdd" />

## Proposed change

Add an option to never merge the spools if a tag is already set (ie. always create).
Result:
<img width="1447" alt="Capture d’écran 2025-03-17 à 00 38 09" src="https://github.com/user-attachments/assets/1a561dde-88db-40b7-a634-47fde924b74f" />

Rdiger-36 commented:
I was quite surprised by how many changes are included in your PR — until I realized that the formatting probably got heavily messed up during the commit.
I really like your ideas and your contribution to this project, and I will adopt them as they are.
I'm very happy that you're helping me improve this project. Thank you so much for that!

Rdiger-36 commented:
I removed the .gitignore as it's not needed for the project itself.
Also updated some information in README.md.

bemble commented:
> I removed the .gitignore as it's not needed for the project itself. Also updated some information in README.md.

Sorry but could you explain why a gitignore is not needed for the project itself? Every time someone wants to contribute, he will have to create this file in order to avoid `node_modules` and other stuff being committed. If there are files you never want to have committed, you add them in the gitignore, and you commit the gitignore, which simplifies everybody's life, no?

bemble commented:
So, yes, it's the practice (that's why I was surprised and why I'm asking 👍). That's the reason why these files exist: being able to systematically exclude files from your git (more info here: https://github.com/github/gitignore).

Rdiger-36 commented:
Yes, I know what it is and what it does. I use it in my local repo. As I said, I never thought about making it public, I just assumed everyone has their own.
